apps/bfd-server/bfd-server-war/src/main/config/server-start.sh

```diff
@@ -167,6 +167,7 @@ BFD_PORT="${serverPortHttps}" \
 	BFD_WAR="${warArtifact}" \
 	"${javaHome}/bin/java" \
 	"${maxHeapArg}" \
+	-Dcapsule.jvm.args="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8083" \
```

Comment:
What's this about?

Reply:
Yeah, is this script used in production? If so it is going to open up a debug port. Not sure that is the desired behavior here.

Reply:
Sorry, this is the "I'm close to done" part. Still working on cleaning that up: want to make sure we keep it around, but disabled, as I've now had to go look up how to do it a few times, and it's getting old. And no, this script is not used in production.

I will be pushing a commit once I get a passing local build.

Reply:
Done. Was trying for something fancier, but Bash gets upset if I add empty variables to the server start command for some reason that I'm too lazy to look up.